base/docs/helpdb/Base.jl

```diff
@@ -923,13 +923,13 @@ Stacktrace:
 BoundsError
 
 """
-    invoke(f, argtypes::Type, args...)
+    invoke(f, argtypes::Type, args...; kwargs...)
 
-Invoke a method for the given generic function `f` matching the specified types `argtypes`, on
-the specified arguments `args`. The arguments must be compatible with the specified types. This
-allows invoking a method other than the most specific matching method, which is useful when
-the behavior of a more general definition is explicitly needed (often as part of the
-implementation of a more specific method of the same function).
+Invoke a method for the given generic function `f` matching the specified types `argtypes` on
+the specified arguments `args` and passing the keyword arguments `kwargs`. The arguments must
+be compatible with the specified types. This allows invoking a method other than the most
```

Comment:
That sounds pretty repetitive with the previous sentence. I'd suggest we keep it   or delete this sentence since the previous sentence is pretty clear already.